docs/application/native/guides/internationalization/i18n.md

```diff
@@ -783,17 +783,17 @@ The UCharIter functions return code unit values in the range of 0 - 0xffff. Befo
 
 The PluralFormat API (in [mobile](../../api/mobile/latest/group__CAPI__BASE__UTILS__I18N__PLURAL__FORMAT__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__BASE__UTILS__I18N__PLURAL__FORMAT__MODULE.html) applications) allows you to [format defined plural words](#plural_format_examples). Alongside with PluralRules module, you can define rules to which format would apply.
 
-Plural Format supports the creation of internationalized messages with plural inflection.
-It is based on plural selection, i.e. the caller specifies messages for each plural case
-that can appear in the user's language and the Plural Format selects the appropriate message
-based on the number.
+Plural Format supports creating locale-dependent messages with plural inflection.
+To enable plural formatting, user have to specify each message for every plural case
+that will be used, based on the desired language. Then the Plural Format module will be
+selecting appropriate text based on provided number.
 
 ### The Problem of Plural Forms in Internationalized Messages:
```

Comment:
Please change as the following:

Problem of plural forms in internationalized messages
Suggest: Can we rephrase the heading to “Issues of plural forms in internationalized messages”.

According to our style guide it is recommended to avoid using the word "problem" in headings.